pkgs/servers/sql/materialize/default.nix

```
@@ -87,11 +87,14 @@ rustPlatform.buildRustPackage rec {
     install --mode=444 -D ./misc/dist/materialized.service $out/etc/systemd/system/materialized.service
   '';
 
+  # The check phase currently fails on aarch64-darwin. This disables
+  # checks for that architecture but leaves them in place for all others.
```

```suggestion
  # tests currently fail on aarch64-darwin:
  # ...
```
Please replace ... with why the tests fail.